src/modules/toolbox/components/toolBar/ToolBar.js

```diff
@@ -52,6 +52,10 @@ export class ToolBar extends MvuElement {
 		this.observe(state => state.network.fetching, fetching => this.signal(Update_Fetching, fetching));
 		this.observe(state => state.media, media => this.signal(Update_IsPortrait_HasMinWidth, { isPortrait: media.portrait, hasMinWidth: media.minWidth }));
 		this.observe(state => state.tools.current, current => this._toolId = current);
+
+		if (this.getModel().isPortrait) {
+			this.signal(Update_IsOpen, false);
+		}
```

Comment:
Found no test on initial presence or absence of `.is-open` class
(should be part of the 'responsive layout' tests, line 213)